src/contributing.md

```diff
@@ -476,27 +476,29 @@ For people new to Rust, and just starting to contribute, or even for
 more seasoned developers, some useful places to look for information
 are:
 
-* The [rustc dev guide] contains information about how various parts of the
+* This guide contains information about how various parts of the
   compiler work and how to contribute to the compiler
 * [Rust Forge][rustforge] contains additional documentation, including
   write-ups of how to achieve common tasks
 * The [Rust Internals forum][rif], a place to ask questions and
   discuss Rust's internals
-* The [generated documentation for rust's compiler][gdfrustc]
-* The [rust reference][rr], even though it doesn't specifically talk about
-  Rust's internals, it's a great resource nonetheless
+* The [generated documentation for Rust's compiler][gdfrustc]
+* The [Rust reference][rr], even though it doesn't specifically talk about
+  Rust's internals is a great resource nonetheless
```

Comment:
```suggestion
  Rust's internals, is a great resource nonetheless
```

Reply:
I wasn't sure about whether there should be a comma, but I guess it's probably better to err on the side of having one :)